crypto/provider_core.c

```diff
@@ -805,6 +809,26 @@ static const OSSL_DISPATCH core_dispatch_[] = {
     { OSSL_FUNC_CORE_PUT_ERROR, (void (*)(void))core_put_error },
     { OSSL_FUNC_CORE_ADD_ERROR_VDATA, (void (*)(void))core_add_error_vdata },
 #endif
+
+    { OSSL_FUNC_CRYPTO_MALLOC, (void (*)(void))CRYPTO_malloc },
```

Comment:
General comment: why isn't there a macro for declaring these entries to reduce boilerplate code? For example, `OSSL_DISPATCH_ENTRY(CRYPTO_MALLOC, CRYPTO_malloc)`?

Reply:
@mattcaswell @levitte

Reply:
I don't mind such a macro.  PR welcome :wink: